packages/library/src/transaction-confirmation.ts

```diff
@@ -73,7 +77,18 @@ export function createDefaultRecentTransactionConfirmer({
     rpc,
     rpcSubscriptions,
 }: DefaultRecentTransactionConfirmerConfig) {
-    const getBlockHeightExceedencePromise = createBlockHeightExceedencePromiseFactory(rpcSubscriptions);
+    const getSleepDelayBetweenBlockHeightFetches = getDefaultSleepDelayBetweenBlockHeightFetchesGetter(
+        400 /* msPerSlot */,
+    );
+    const getSleepDelayBetweenBlockHeightFetchFailures = getDefaultSleepDelayBetweenBlockHeightFetchFailuresGetter(
+        5000 /* minDelay */,
```

Comment:
This is a *big* handwave. The thinking here was ‘if we failed to fetch a blockhash, we might as well be conservative and give the RPC a break.’ The worst case scenario for this code failing forever is your transaction confirmation never ‘blockheight-ing out.’